test/io-tests.sh

```diff
@@ -273,13 +273,10 @@ replaceConfigValue(){
 }
 
 getSocketStatus() {
-  curl -sL -w "%{http_code}\\n" -o /dev/null localhost:54321
+  curl -sL -w "%{http_code}\\n" -o /dev/null --unix-socket /tmp/postgrest.sock http://localhost/
```

Comment:
this will now fail with curl < 7.40, which does not support --unix-socket.. but is 5 years old. Should be fine, I guess.